.github/workflows/formatting.yml

```diff
@@ -0,0 +1,37 @@
+name: Formatting
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  format:
+    name: black
+    runs-on: ubuntu-latest
+    
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Run black
+        uses: rickstaa/action-black@v1
```

Comment:
Instead of using this action created by this user, can we use the official psf black action? It says so in the [README](https://github.com/rickstaa/action-black#action-black-github-action) for this action as well.

Also, could you pin `black`'s version to `22.10.0` like we had in the previous action? That would make formatting a little more consistent.

Reply:
<img width="892" alt="Screen Shot 2022-10-13 at 8 59 58 PM" src="https://user-images.githubusercontent.com/67194380/195633167-5de24f6d-ca8a-4dd9-b5c8-0b624b9f1e73.png">

@hrushikeshrv I've made some changes to the workflow while also pinning the version with the original GitHub Actions provided by psf, but now I'm getting this error. Thoughts?

Reply:
@hrushikeshrv Okay turns out, I was using the unstable versions. Now I've switched the runner to Windows and the version to [black's stability policy](https://black.readthedocs.io/en/stable/the_black_code_style/index.html#labels-stability-policy) and it works.